index.html

```diff
@@ -260,41 +245,55 @@ <h3>
 service
             </li>
             <li>
-creating the DID document JSON-LD file including a suitable keypair, e.g.,
-using the Koblitz Curve, and storing the <code>did.json</code> file under the
-well-known URL to represent the entire domain, or under the specified path if
-many DIDs will be resolved in this domain.
+creating the DID document JSON, and storing the file as the index of the
```

Comment:
re "storing the file as the index" -- we don't define what we mean by "index" anywhere.

Reply:
I am referring to the "directory index'. It has been commonplace for webservers to return index.html or some other configured. Nginx uses `index` and Apache uses `DirectoryIndex` to set a file or files to return when a user requests a directory path.

I can add a description.

Reply:
Thanks. Adding a description would be helpful (so that people don't think we mean `index.html`)

Reply:
`index.html` has been a common default as long as I can remember, but it is possible to set other files as the index.

Does this do it?
db763a18f0242fdbc9894fb28e98c9fd9e180a3f